src/Amadevus.RecordGenerator.Generators/RecordDescriptor.cs

```diff
@@ -7,14 +7,17 @@ namespace Amadevus.RecordGenerator.Generators
 {
     internal partial class RecordDescriptor
     {
-        public RecordDescriptor(TypeSyntax Type, SyntaxToken TypeIdentifier, ImmutableArray<Entry> Entries, TypeDeclarationSyntax TypeDeclaration)
+        public RecordDescriptor(Features features, TypeSyntax Type, SyntaxToken TypeIdentifier, ImmutableArray<Entry> Entries, TypeDeclarationSyntax TypeDeclaration)
         {
             this.Type = Type;
             this.TypeIdentifier = TypeIdentifier;
             this.Entries = Entries;
             this.TypeDeclaration = TypeDeclaration;
+            Features = features;
         }
 
+        public Features Features { get; }
```

Comment:
I don't think this belongs here because it feels like this is mixing description of a record with code generation options.

Reply:
> I don't think this belongs here because it feels like this is mixing description of a record with code generation options.

[my closing review comment](https://github.com/amis92/RecordGenerator/pull/66#pullrequestreview-267179972) says more.

Reply:
Let's move it to a new issue.

Reply:
See #71